tests/test_cmd_build.py

```diff
@@ -1,11 +1,54 @@
 """Tests `oasis build`."""
 
+import os
 import os.path as osp
+import shutil
 from subprocess import PIPE
 
 
-def test_cargo_version(oenv, mock_tool):
-    mock_tool.create_at(osp.join(oenv.bin_dir, 'cargo'))
-    svc_dir = osp.join(oenv.create_project(), 'service')
-    cp = oenv.run('oasis build', cwd=svc_dir, stdout=PIPE)
-    assert mock_tool.parse_output(cp.stdout)[0]['args'][0] == '+nightly-2019-08-26'
+def test_build_multiproj(oenv, temp_dir):
+    multiproj_dir = osp.join(temp_dir, 'multiproj')
+    target_dir = osp.join(multiproj_dir, 'target', 'service')
+
+    shutil.copytree(osp.join(osp.dirname(__file__), 'res', 'multiproj'), multiproj_dir)
+
+    # test no repo
```

Comment:
these looks like a sequence of independent tests. If that's the case, would it make sense to keep them separate?

Reply:
It would, but the testing framework is stateless, so it creates a new `oenv` for each test. I'd like to cache the build results, so they run in the same oenv.